cli/cache.rs

```diff
@@ -350,3 +340,195 @@ impl CacherLoader for MemoryCacher {
     self
   }
 }
+
+fn get_cache_filename(url: &Url) -> Option<PathBuf> {
```

Comment:
This function and the one below were moved from `disk_cache.rs` into here. I didn't change the contents, only moved them off the struct and made it private here.